.github/workflows/test.yml

```
@@ -0,0 +1,41 @@
+          - "3.13"
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Start up containers
+        run: |
+          docker compose -f ./test_containers/compose.yaml up --detach --wait --wait-timeout 600
+
+      - name: Setup Python
+        uses: actions/setup-python@v4
```

actionlint (1.7.4)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)